README.md

```diff
@@ -5,25 +5,37 @@
 Package kayvee provides methods to output human and machine parseable strings,
 with a "json" format.
 
-## [Logger Documentation](./logger)
+## [Logger API Documentation](./logger)
 
```

Comment:
I'm blind, it's already there.  I thought this was in infra-docs for some reason.